Makefile

```diff
@@ -157,3 +158,74 @@ endif
 	docker push ${REGISTRY}/karmada-scheduler-estimator:${VERSION}
 	docker push ${REGISTRY}/karmada-interpreter-webhook-example:${VERSION}
 	docker push ${REGISTRY}/karmada-aggregated-apiserver:${VERSION}
+
+# Build and push multi-platform image to DockerHub
+mp-image-karmada-controller-manager: karmada-controller-manager
+ifneq ($(REGISTRY_USER_NAME), "")
+	docker login -u ${REGISTRY_USER_NAME} -p ${REGISTRY_PASSWORD}
+endif
```

Comment:
Maybe we can't do the login operation here, it's not safe to dump the password out.

Reply:
Seems the GitHub Action won't dump the variables from secrets.

Reply:
https://github.com/karmada-io/karmada/runs/5594931141?check_suite_focus=true#step:4:185